Automattic/delphin
# Fix animating the synonyms

This PR updates `Synonyms` to animate when it opens and closes.

![synonyms](https://cloud.githubusercontent.com/assets/1130674/15406388/240fcdf2-1dbb-11e6-8905-cb3968eefa5f.gif)

**Testing**
- Visit `/search?q=query`
- Click on a keyword
- Assert that the synonyms animate as they appear.
- Click on the keyword again
- Assert that the synonyms animate as they disappear.
